gnucash/gnucash.cpp

```diff
@@ -195,9 +195,9 @@ scm_run_gnucash (void *data, [[maybe_unused]] int argc, [[maybe_unused]] char **
 
     gnc_hook_run(HOOK_STARTUP, NULL);
 
-    boost::optional<boost::filesystem::path> fn;
     if (!user_file_spec->nofile)
     {
+        boost::optional<boost::filesystem::path> fn;
         if (auto fn = get_file_to_load (user_file_spec->file_to_load))
```

Comment:
You redefine fn inside the `if` clause in the next line. I think you can drop this declaration.

Reply:
Ah thanks for the tip—maybe that explains the mysterious CI failure.